tests/cpp/test_module_fallback.cpp

```diff
@@ -1,74 +0,0 @@
-#include <string>
```

Comment:
Does this not pass with cos similarity?

Reply:
No I moved this to python to compare with nn.Module. Check out tests/api/test_module_fallback.py

Reply:
We still need to make sure this c++ api works

Reply:
So a question - In that case, we will compare torch.jit.script result with TRT engine result which is okay ? The reason I ask is we saw even torch.jit.script results differed from nn.Module in terms of correctness (not cosine sim) so I thought a proper comparison will only be with nn.Module

Reply:
added it back